exercises/queen-attack/canonical-data.json

```diff
@@ -18,39 +18,54 @@
           "description": "queen with a valid position",
           "property": "create",
           "queen": {
-            "position": "(2,2)"
+            "position": {
+              "file": 2,
+              "rank": 2
```

Comment:
The [description](https://github.com/exercism/problem-specifications/blob/master/exercises/queen-attack/description.md) uses the terms `row` and `column` I think we should stick with those.

Reply:
Agree with @Insti, we should follow what is in the description (or else change the description).

Reply:
I'd argue against changing the description.

Reply:
I agree, row/column works great for me. Made the change.